src/planner/binder/query_node/bind_select_node.cpp

```diff
@@ -461,6 +462,9 @@ unique_ptr<BoundQueryNode> Binder::BindNode(SelectNode &statement) {
 			result->groups.group_expressions.push_back(std::move(expr));
 			expr = std::move(group_ref);
 		}
+		if (!result->unnests.empty() && is_window && is_window_aggregate) {
```

Comment:
`result->unnests` contains the unnests for the entire query - this means that unnests and window functions cannot be mixed. For example, will this query still work:

```sql
select unnest([5, 4, 3]), sum(41) over ();
```